components/common/layout.tsx

```diff
@@ -0,0 +1,28 @@
+import FooterComponent from "./footerComponent/footerComponent";
+import NavBarComponent from "./navBarComponent/navBarcomponent";
+import { useState } from "react";
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+export default function Layout({ children }: LayoutProps): JSX.Element {
```

Comment:
Use a consistent format to export.

Either
```
export default function
```

 or 
 
 ```
 function Layout...
 
 export default Layout;
 ```

Reply:
Fixed